scripts/trufflehog-check.sh

```diff
@@ -15,19 +15,30 @@ fi
 
 echo "trufflehog-check.sh: Scanning repo ..."
 
-last_merge=$(git log --format=format:"%H" -n 1 raft-tdp-main)
-
 # $1 - The first argument to this script, the current git branch name
+CIRCLE_BRANCH=$1
+
+# If the current PR branch includes `raft-tdp-main`, it a merge against HHS:main,
+# so we update the target branch to scan against.
+if [ "$CIRCLE_BRANCH" == "raft-tdp-main" ] ; then
+  TARGET=main
```

Comment:
This will set the correct target branch to scan against when merging into raft-tdp-main.